### PR TITLE
Admin adviser experience (AAE)

### PR DESCRIPTION
**Pull request by jasonblais**
#### Summary
This PR adds admin adviser experience codes to five existing messages.

There are no functional changes. More AAE codes are in the works.

#### Ticket Link
None.

Spec: https://docs.google.com/document/d/1SnpFASzQ4ADa3qzibYJyHNUhrvRQNNIvIVrUL71BCvo

Docs PR: https://github.com/mattermost/docs/pull/3671

### CODE REVIEW

**Comment by jespino** [comment]
You are adding the references but you are not linking them anyway, looks confusing to have that codes there without any explanation.

File: components/announcement_bar/configuration_bar/configuration_bar.jsx [hunk 1]
```diff
@@ -119,9 +121,11 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
         ) {
             const emailMessage = formatMessage({
                 id: AnnouncementBarMessages.PREVIEW_MODE,
-                defaultMessage: 'Preview Mode: Email notifications have not been configured',
+                defaultMessage: 'Preview Mode: Email notifications have not been configured. [AAE-000-0011](!{link})',
+                values={{
+                    link: AAE_REGISTRY_LINK,
+                }}
             });
-
```
Comment (jasonblais) [inline components/announcement_bar/configuration_bar/configuration_bar.jsx:124]: @jespino I tried adding the link here, but I am quite certain I'm using the `values` field incorrectly here, causing the build to break. Same below. Would you have an example of how I should approach updating this constant?

**Comment by jasonblais** [comment]
@jespino That's correct, the link will work once https://github.com/mattermost/docs/pull/3671 is merged.

Do you know why the build fails if the code looks good? I'm having trouble figuring out based on the build logs.

**Comment by jespino** [comment]
@jasonblais yes, probably some trailing comma or something like that, let me fix that for you :)

**Comment by jespino** [comment]
@jasonblais fixed

**Comment by jasonblais** [comment]
@jespino Thank you :) 🙌

**Comment by jasonblais** [comment]
@srkgupta Given this involved more code changes than I usually do, it would be good to verify the link is rendered correctly for the announcement bar when
 - site URL is not set 
 - SMTP is not configured
 - Enterprise license is expiring

**Comment by jasonblais** [comment]
@srkgupta @catalintomai on the above, cc @esadur

**Comment by srkgupta** [comment]
@jasonblais my apologies. Somehow this missed my radar because I had already approved the PR. I will check this again, but can you please resolve the merge conflicts so that I can retest this. Thanks.

**Comment by jasonblais** [comment]
@srkgupta Done

**Comment by jasonblais** [comment]
Assigning to Katie and Dennis to determine next steps.

**Comment by srkgupta** [comment]
Hi @jasonblais 

The link you had added in the PR is not pointing to the correct URL. For example, its showing up as http://localhost:8065/admin_console/environment/!https://mattermost.com/pl/default-aae-message-code-registry on my localhost instance

**Comment by wiersgallak** [comment]
@jasonblais Wondering if want to close this PR for now since we have paused Admin Advisor?

**Comment by jasonblais** [comment]
@wiersgallak Let's close. We'll need to revisit this feature at a later time.

**Pull request closed without merging**